### PR TITLE
{vis}[GCC/10.2.0,GCCcore/10.2.0,iccifort/2020.4.304] POV-Ray v3.7.0.8, SDL2 v2.0.14

### PR DESCRIPTION
**Pull request by schiotz**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/17ff1b51ef4ed9286a29707704847bcb for a full test report.

**Comment by Micket** [comment]
Going in, thanks @schiotz!

File: easybuild/easyconfigs/p/POV-Ray/POV-Ray-3.7.0.8-iccifort-2020.4.304.eb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    ('zlib', '1.2.11'),
+    ('libpng', '1.6.37'),
+    ('libjpeg-turbo', '2.0.5'),
+    ('X11', '20201008'),
+    ('LibTIFF', '4.1.0'),
+    ('SDL2', '2.0.14'),
+]
+
+builddependencies = [
+    ('binutils', '2.35'),
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/POV-Ray/POV-Ray-3.7.0.8-iccifort-2020.4.304.eb:46]: Doesn't need binutils at GCC/iccifort level

And please move builddeps ahead of deps.

File: easybuild/easyconfigs/p/POV-Ray/POV-Ray-3.7.0.8-GCC-10.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    ('zlib', '1.2.11'),
+    ('libpng', '1.6.37'),
+    ('libjpeg-turbo', '2.0.5'),
+    ('X11', '20201008'),
+    ('LibTIFF', '4.1.0'),
+    ('SDL2', '2.0.14'),
+]
+
+builddependencies = [
+    ('binutils', '2.35'),
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/POV-Ray/POV-Ray-3.7.0.8-GCC-10.2.0.eb:46]: Doesn't need binutils at GCC/iccifort level

And please move builddeps ahead of deps.

**Comment by akesandgren** [comment]
Oops, was already merged sorry. But please make a new PR to fix this...

**Comment by schiotz** [comment]
> Oops, was already merged sorry. But please make a new PR to fix this...

Done.  PR #12342

**Pull request merged**